### DISCUSSION TITLE
Needed: No logo at all

### DISCUSSION THREAD

**Discussion by kora24** [fastfetch-cli/fastfetch#1023] [Q&A] [answered]
Hello,

Here is my config.jsonc

`{
"$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
"modules": [
"break",
"title",
"separator",
"os",
"host",
"kernel",
"packages",
"shell",
"display",
"de",
"wm",
"wmtheme",
"theme",
"icons",
"font",
"cursor",
"terminal",
"terminalfont",
"cpu",
"gpu",
"memory",
"swap",
"disk",
"battery",
"poweradapter",
"break",
"break"
]
}`

And the big distro logo to the left. How can I get rid of the logo? I just want it to be the text and nothing else.

Thank you.

**Comment by nyaff** [comment, marked as answer]
Add `"logo": null,` before "modules" to get rid of the logo:

`{ "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json", "logo": null, "modules": [ "break", "title", "separator", "os", "host", "kernel", "packages", "shell", "display", "de", "wm", "wmtheme", "theme", "icons", "font", "cursor", "terminal", "terminalfont", "cpu", "gpu", "memory", "swap", "disk", "battery", "poweradapter", "break", "break" ] }`

**Comment by kora24** [comment]
Thank you very much!

**Comment by JamesClarke7283** [comment]
FYI, for users who came here from a web search, for the commandline way of doing this its:
```sh
fastfetch  -l none
```

**Reply by roke-julian-lockhart** [reply]
Can the palette be removed, too? It's for [`discussion.fedoraproject.org/t/125742/35`][1].

[1]: https://discussion.fedoraproject.org/t/we-miss-kinfo-for-gnome/125742/35

**Reply by CarterLi** [reply]
`fastfetch --format json` is the best way to do it, I think.

**Reply by roke-julian-lockhart** [reply]
@CarterLi, `fastfetch --format json | yq -P` [^1] gets me <em>closer</em>, `kinfo`'s output, but it still includes the "Colors" section, which is what I care about most:

[^1]: [`stackoverflow.com/revisions/62798787/6`][1]

[1]: https://stackoverflow.com/revisions/62798787/6#:~:text=cat%20%22$DEFAULTS_FILE%22-,%7C%20yq%20%2DP,-%23%20or%20yq

> ~~~YAML
> - type: Colors
>   error: Unsupported for JSON format
> ~~~

It's also a little verbose, because the keys are themselves values of `type`, rather than keys.
